CONTRIBUTING.md

```diff
@@ -0,0 +1,84 @@
+# How to contribute
+
+First off, thanks for taking the time to contribute to AZ Quickstart! 
+
+AZ Quickstart is created by [Arizona Digital](https://digital.arizona.edu/), a
+team of web-focused volunteers that meet weekly to create projects like [Arizona Bootstrap](https://digital.arizona.edu/ua-bootstrap) and 
+[Arizona Quickstart](https://quickstart.arizona.edu/).
+
+## Things you'll need to get started
+
+  * A GitHub [account](https://github.com/join).
+  * [Slack](https://uarizona.slack.com) is our main source of communications.
+    * Use the `#ua-quickstart-d8` channel for questions/comments related to this
+      project.
+    * Use the `#friday-meetings` channel to ask questions or get updates related 
+      to Arizona Digital meetings, both physical and via Zoom.
+    * Use the `#uadigital-general` channel to ask general questions related to
+      Arizona Digital.
+  * A basic understanding of [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+  * [Lando](https://docs.lando.dev/basics/installation.html) for local development
+
+## Submitting a bug/issue/feature request
+
+### Security Issues
+
+If you found a security vulnerability and it's related to Drupal core or a 
+Drupal contrib module, please follow 
+[these](https://www.drupal.org/drupal-security-team/security-team-procedures/drupal-security-team-disclosure-policy-for-security) instructions.
+
+If it's a security issue related to `az_quickstart` code, please email us here:
+az-digital@web.arizona.edu
+
```

Comment:
Got some feedback from ISO about this. Paul advised us to consult our ISRM surrounding the risk involved with accepting security reports via email. My opinion as the ISRM for Arizona Digital is that the risk of not providing an easy, universal, way to report an issue outweighs the risk of that report being discovered by someone gaining access to our private slack channel which the security email address feeds into. As such, we are accepting this risk in order to mitigate the greater risk of someone choosing to not report a vulnerability they have discovered in our codebase. Utilizing slack instead of a shared inbox also increases the timeliness of our response to the incoming security issue report which may be critical depending on the issue severity.

Reply:
I think it may be best to use a dedicated security address for this though, something like `security@digital.arizona.edu` if possible.

Reply:
We've decided to merge now and update the email address as a follow up issue